src/Main.java

```diff
@@ -1,9 +1,16 @@
 
 public class Main {
     public static void main(String[] args) {
+
+        //instancia de ejemplo
         Animal a1 = new Dog();
         a1.makeSound();
 
+
+        //instancia propia
+        Animal a2 = new Cow();
```

Comment:
Please use descriptive name for the variables. Something like `Animal cow = new Cow();`